admin_manual/configuration/files/encryption_configuration_quick_guide.rst

```diff
@@ -1,73 +1,111 @@
+**Will be removed in future releases**.
+
+**Master Key**
+~~~~~~~~~~~~~~
+- The **recommended** type of encryption.
+- Best to activate on new instances with no data.
+- If you have existing data, use **encrypt all** command. Depending on the amount of existing data, this operation can take a long time.
+
+**Activation**
+
+::
+	occ maintenance:singleuser --on
+	occ app:enable encryption
+	occ encryption:enable
+	occ encryption:select-encryption-type masterkey -y
+	occ encryption:encrypt-all
+	occ maintenance:singleuser --off
+
+**Status**
+
+::
+
+	occ encryption:status 
 
-- **Master Key:** there is only one key (or key pair) and all files are encrypted using that key pair.
-  
-How To Enable Encryption
-------------------------
+**Decryption**
 
-How to enable **Master Key** encryption:
+Depending on the amount of existing data, this operation can take a long time.
+::
+	occ maintenance:singleuser --on
+ 	occ encryption:decrypt-all
+ 	occ maintenance:singleuser --off
+
+**Deactivation**
+
+::
+	occ encryption:disable
+	# ignore the "alredy disabled" message
+	occ app:disable encryption
+  	
+If the master key has been compromised or exposed, you can recreate it. You will need the current master key for it.
 
 ::
 
-  occ app:enable encryption
-  occ encryption:enable
-  occ encryption:select-encryption-type masterkey
-  occ encryption:encrypt-all
+	occ encryption:recreate-master-key 
+
+
+**User-specific Key**
+~~~~~~~~~~~~~~~~~~~~~
 
-How to enable **User-specific Key** encryption:
+**Activation**
 
 ::
 
-  occ app:enable encryption
-  occ encryption:enable
-  occ encryption:select-encryption-type user-keys
-  occ encryption:encrypt-all 
+	occ maintenance:singleuser --on
+	occ app:enable encryption
+	occ encryption:enable
+	occ encryption:select-encryption-type user-keys
+	occ encryption:encrypt-all
+	occ maintenance:singleuser --off 
 
 
-After User-specific encryption is enabled, users must log out and log back in to trigger the automatic personal encryption key genaration process. 
+After User-specific encryption is enabled, users must log out and log back in to trigger the automatic personal encryption key generation process. 
 
-How To Enable Users File **Recovery Keys**
-------------------------------------------
+**Recovery Key**
 
-Go to the "Encryption" section of your Admin page and set a recovery key password. You then need to ask the users to opt-in to the Recovery Key. If a user decides not to opt-in to the recovery key and forgets about his password, all of the users data can not be decrypted anymore.
+- Go to the "Encryption" section of your Admin page
+- Set a Recovery Key password
+- Ask the users to opt-in to the Recovery Key
+
+If a **user** decides **not to opt-in** to the Recovery Key and forgets/loses his password, **all of the user's data can not be decrypted anymore**. This leads to **permanent data loss**.
```

Comment:
Isn't this true, if recovery key is set by the admin and not enabled by the user in his/her Personal page?

Reply:
Yes. That's the opt-in. Isn't it?

Was this your question?

Reply:
Yes. That was my question. Thanks for clarification.